src/libs/ReportScrollManager/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export const flatListRef = React.createRef();
```

Comment:
This goes against our [style guide for refs](https://github.com/Expensify/Expensify.cash/blob/main/STYLE.md#use-refs-appropriately), but I see why it's necessary in this case. So I think we should add a comment to explain that this ref is created using `React.createRef` here because this function is used by a component that doesn't have access to the original ref.